pcsx2/CDVD/CDVDaccess.h

```diff
@@ -168,3 +175,17 @@ extern s32 DoCDVDreadTrack(u32 lsn, int mode);
 extern s32 DoCDVDgetBuffer(u8* buffer);
 extern s32 DoCDVDdetectDiskType();
 extern void DoCDVDresetDiskTypeCache();
+
+static std::vector<cdvdSubQ> m_tracks;
+static std::vector<std::pair<u32, cdvdSubQ>> indices;
+static std::optional<cdvdSubQ> zero_pregap;
+
+static const _cdvdSubQ* GetIndex(u32 n)
+{
+	for (const auto& it : indices)
+	{
+		if (it.first == n)
+			return &it.second;
+	}
+	return nullptr;
+}
```

Comment:
Why is this not in the cue parser with the rest of the cue parser code since that's the only place that calls it?